src/room/useGroupCall.ts

```diff
@@ -299,6 +300,13 @@ export function useGroupCall(groupCall: GroupCall): UseGroupCallReturnType {
     groupCall.setMicrophoneMuted(!groupCall.isMicrophoneMuted());
   }, [groupCall]);
 
+  const setVoiceActivityThreshold = useCallback(
+    (value: number) => {
+      groupCall.localCallFeed?.setVoiceActivityThreshold(value);
```

Comment:
Hmm, I think we should wrap the `setVoiceActivityThreshold()` method by a method on `GroupCall`